rules/retention_rules.yaml

```diff
@@ -0,0 +1,44 @@
+# Copyright 2018 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# rules:
+#   - name: Buckets in Organization must have a retention between 180 and 730 days.
+#     applies_to:
+#       - bucket
+#     resource:
+#       - type: organization
+#         resource_ids:
+#           - "123456789012"
+#     minimum_retention: 180
+#     maximum_retention: 730
+#   - name: Buckets in Project must have a retention between 100 and 365 days.
```

Comment:
It might be good to have for the 3 examples in this file:
1 with a min and a max
1 with min only
1 with max only